_issues/but-hillary-wants-my-guns.md

```diff
@@ -4,32 +4,22 @@ title: "But Hillary wants my guns!"
 categories: guns, gun control
 ---
 
-Q: But Hillary wants my guns!
+Q: Doesn't Hillary want to take away my right to own a gun?
 
 
-A: That's not precisely correct!
+A: Hillary respects law-abiding citizens’ 2nd Amendment right to own guns. She just wants to put in place [common-sense solutions](https://www.hillaryclinton.com/issues/gun-violence-prevention/) to make our communities safer. 
 
-Basically, she doesn't want your guns, unless you're a felon, a
-domestic abuser, or someone with severe mental illnesses. And I think
-we can all agree that people like that shouldn't really have access to
-guns.
+Guns kill [around 33,000](https://en.wikipedia.org/wiki/Gun_violence_in_the_United_States) Americans a year. Hillary is committed to reducing the epidemic of gun violence. 
```

Comment:
Let's not use Wikipedia as a source. Here's a 538 article which has the same statistic: http://fivethirtyeight.com/features/gun-deaths/